client/src/css/Grace.css

```diff
@@ -0,0 +1,74 @@
+.banner {
+  width: 100%;
+  height: 400px;
+  margin-bottom: 20px;
+}
+
+.header {
+  text-align: center;
+  color: #fdf1e8;
+  font-size: 80px;
+  position: absolute;
+  top: 150px;
+  left: 0;
+  width: 100%;
+}
+
+.intro {
+  text-align: center;
+  color: #fdf1e8;
+  font-size: 20px;
+  position: absolute;
+  top: 250px;
+  left: 0;
+  width: 100%;
+  padding: 10px;
+}
```

Comment:
You should use margin-top instead of top in this case
see [this discussion](https://stackoverflow.com/questions/4036176/css-top-vs-margin-top) for more info